modules/installation-azure-config-yaml.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-customizations.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
+
+ifeval::["{context}" == "installing-azure-network-customizations"]
+:with-networking:
+endif::[]
+ifeval::["{context}" != "installing-azure-network-customizations"]
+:without-networking:
+endif::[]
+
+[id="installation-azure-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for Azure
+
+You can customize the `install-config.yaml` file to specify more details about
+your {product-title} cluster's platform or modify the values of the required
+parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your
+`install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2>
+  hyperthreading: Enabled <3> <4>
+  name: master
+  platform:
+    azure:
```

Comment:
@sjug, do you have information about optimizing etcd for Azure? You provided the review for Ashley for the AWS equivalent in 4.1.

Reply:
Currently the only machinepool exposed instance configs are zone, disk size and instance type. So only instance type will potentially affect the etcd performance. It doesn't look like Azure instances decouple the storage and the rest of the instance type, so the AWS recommendations don't translate over exactly.

The disk specifications of the `Standard_DS5_v2` instance is closest to what we recommend for AWS.